experts/MAP.md

```diff
@@ -0,0 +1,4 @@
+# Compiler team expert map
```

Comment:
```suggestion
# Expert Map
```

I don't think we need to say "compiler team", since that is implied by this repository.